chia/types/blockchain_format/proof_of_space.py

```diff
@@ -41,8 +41,13 @@ def verify_and_get_quality_string(
     constants: ConsensusConstants,
     original_challenge_hash: bytes32,
     signage_point: bytes32,
+    *,
+    height: uint32,
 ) -> Optional[bytes32]:
-    # Exactly one of (pool_public_key, pool_contract_puzzle_hash) must not be None
+    """
+    Exactly one of (pool_public_key, pool_contract_puzzle_hash) must not be None
+    If height is passed as None, plot filter validation will not be performed
+    """
```

Comment:
```suggestion
    # Exactly one of (pool_public_key, pool_contract_puzzle_hash) must not be None
```

This whole diff here should be reverted it seems now after the recent changes?

Reply:
Thanks, taken.